buildkite/bazelci.py

```diff
@@ -1352,7 +1366,8 @@ def execute_commands(
             bazel_binary,
             build_only,
             test_only,
-            requested_working_dir,
+            os.getcwd(),
+            PrepareRepoInCwd,
```

Comment:
nit: I'm not very familiar with passing a function as an argument, I believe this function depends on `task_config` in the context? It is a bit hard for me to understand, is it possible to avoid using this pattern?

Reply:
yeah, a closure. That's nothing uncommon to use in Python

Reply:
Okay, then it's all good